src/components/CreateStamp.js

```diff
@@ -25,6 +25,13 @@ class CreateStamp extends Component {
         button.style.display = 'none';
       }
     }
+    const style = {
```

Comment:
Can you give a description of what this is trying to solve ? I'm trying to verify, but I'm not entirely sure what I should be looking out for.

Reply:
Oh wait.. based on the screenshots, this is to deal with the autocomplete for when you type a non-existing timezone ???

Reply:
Yes :) at least the `noOptionsMessage` part; the `dropdownIndicator` bit is also for improving contrast in the, well, drop-down indicator :sweat_smile:

Sorry if that wasn't clear. Should I add code comments to make the intent explicit?

Reply:
I have added a comment in the new version of the branch: [00bc8b4..8ffe6e7](https://github.com/hartman/zonestamp/compare/00bc8b400744e1902464a304a805c2b4d0f3b342..8ffe6e7ee1c3743fd66ad5b7a90f007ae56cea9d).